tap-snapshots/test-v8-to-istanbul.js-TAP.test.js

```diff
@@ -1870,358 +1870,10 @@ Object {
-      },
-      "locations": Array [
-        Object {
-          "end": Object {
-            "column": 10,
-            "line": 31,
-          },
-          "start": Object {
-            "column": 10,
-            "line": 31,
-          },
-        },
-      ],
-      "type": "branch",
-    },
-    "6": Object {
-      "line": 1,
-      "loc": Object {
-        "end": Object {
-          "column": 16,
-          "line": 2,
-        },
-        "start": Object {
-          "column": 0,
-          "line": 1,
-        },
-      },
-      "locations": Array [
-        Object {
-          "end": Object {
-            "column": 16,
-            "line": 2,
-          },
-          "start": Object {
-            "column": 0,
-            "line": 1,
-          },
-        },
-      ],
-      "type": "branch",
-    },
-    "7": Object {
-      "line": 1,
-      "loc": Object {
-        "end": Object {
-          "column": 6,
-          "line": 2,
-        },
-        "start": Object {
-          "column": 0,
-          "line": 1,
-        },
-      },
-      "locations": Array [
-        Object {
-          "end": Object {
-            "column": 6,
-            "line": 2,
-          },
-          "start": Object {
-            "column": 0,
-            "line": 1,
-          },
-        },
-      ],
-      "type": "branch",
-    },
-    "8": Object {
-      "line": 2,
-      "loc": Object {
-        "end": Object {
-          "column": 10,
-          "line": 2,
-        },
-        "start": Object {
-          "column": 0,
-          "line": 2,
-        },
-      },
-      "locations": Array [
-        Object {
-          "end": Object {
-            "column": 10,
-            "line": 2,
-          },
-          "start": Object {
-            "column": 0,
-            "line": 2,
-          },
-        },
-      ],
-      "type": "branch",
-    },
-    "9": Object {
-      "line": 20,
-      "loc": Object {
-        "end": Object {
-          "column": 10,
-          "line": 30,
-        },
-        "start": Object {
-          "column": 12,
-          "line": 20,
-        },
-      },
-      "locations": Array [
-        Object {
-          "end": Object {
-            "column": 10,
-            "line": 30,
-          },
-          "start": Object {
-            "column": 12,
-            "line": 20,
-          },
-        },
-      ],
-      "type": "branch",
-    },
-  },
-  "f": Object {
-    "0": 1,
-  },
-  "fnMap": Object {
-    "0": Object {
-      "decl": Object {
-        "end": Object {
-          "column": 10,
-          "line": 30,
-        },
-        "start": Object {
-          "column": 12,
-          "line": 20,
-        },
-      },
-      "line": 20,
-      "loc": Object {
-        "end": Object {
-          "column": 10,
-          "line": 30,
-        },
-        "start": Object {
-          "column": 12,
-          "line": 20,
-        },
-      },
-      "name": "e",
-    },
-  },
+  "b": Object {},
+  "branchMap": Object {},
```

Comment:
this seems weird to me, I wouldn't have expected us to need to update the snapshot for the non-multi-source use-case, and it looks like this updates the snapshot with empty information.

Reply:
@bcoe this was indeed a bug, I fixed it